iron-form.html

```diff
@@ -383,6 +384,7 @@
             var defaults = this._defaults.get(node);
             node.value = defaults.value;
             node.checked = defaults.checked;
+            node.invalid = defaults.invalid;
```

Comment:
Do you think we should check if these exist? like if `defaults` _has_ an `invalid`/`checked` property, only then set it? Otherwise we're technically creating these properties that don't exist. It may be overthinking it

Reply:
Good point, better doing less - will change this

Reply:
yeah, perfect